gstd/src/msg/async.rs

```diff
@@ -61,6 +62,15 @@ impl<D: Decode> FusedFuture for CodecMessageFuture<D> {
     }
 }
 
+/// To interrupt a program execition waiting for a reply on a previous message,
+/// one need to specify an `.await` expression.
```

Comment:
probably `to call .await` would be more suitable

Reply:
ok